gufe/tests/test_custom_json.py

```diff
@@ -113,15 +113,7 @@ def setup_method(self):
             np.array([1, 0]),
             np.array([1.0, 2.0, 3.0], dtype=np.float32),
         ]
-        shapes = [
-            [2, 2],
-            [
-                2,
-            ],
-            [
-                3,
-            ],
-        ]
+        shapes = [[2, 2], [2], [3]]
```

Comment:
maybe the auto-formatter will revert this, but it was hurting my brain.

Reply:
Oh shoot, yeah I hope the autoformatter gives something more reasonable!

Reply:
```suggestion
        shapes = [[2, 2], [2], [3]]  # fmt: skip
```
That way we don't have to rely on the formatter being kind.